src/app/main.ml

```diff
@@ -150,6 +151,7 @@ let begin_program
         Usbmux.Logging.(
           let relay_with =
             R.make_tunnels
+              ~bind_host:bind_host
```

Comment:
I tried to make changes that  you suggest and get compilation error.
File "src/lib/usbmux.ml", line 551, characters 19-31:
Error: This expression has type string option
       but an expression was expected of type string
Please help me to understand what to change.